adapters/telegram/src/polling.ts

```diff
@@ -83,7 +83,7 @@ export namespace HttpPolling {
 
   export const Config: Schema<Config> = Schema.object({
     protocol: Schema.const('polling' as const).required(),
-    pollingTimeout: Schema.natural().role('time').default(Time.minute).description('通过长轮询获取更新时请求的超时 (单位为秒)。'),
+    pollingTimeout: Schema.natural().role('ms').default(Time.second * 25).description('通过长轮询获取更新时请求的超时 (单位为毫秒，这里使用 1000 的倍数)。'),
```

Comment:
```suggestion
    pollingTimeout: Schema.natural().role('ms').default(Time.second * 25).description('通过长轮询获取更新时请求的超时 (单位为毫秒)。'),
```

No no no nope, you are misdirecting people.
The user SHOULD write `1000` to indicate one second, what I said above is meaning that you should keep it simple for code contributors or maintainers, not for a regular user.

Also, you should change the `pollingTimeout / Time.second` back to `pollingTimeout / 1000` as before.